include/remill/Arch/Runtime/Intrinsics.h

```diff
@@ -68,7 +69,7 @@ __remill_write_memory_f32(Memory *, addr_t, float32_t);
 __remill_write_memory_f64(Memory *, addr_t, float64_t);
 
 [[gnu::used]] extern Memory *__remill_write_memory_f80(Memory *, addr_t,
-                                                       float80_t);
+                                                       native_float80_t);
```

Comment:
`const native_float80_t &`

Reply:
Maybe? Dunno if this actually matters.

Reply:
I will make the changes but it should not matter. It will require additional fixes in Lower memory op pass to handle this variant of write intrinsics.